include/swift/AST/ASTTypeIDZone.def

```diff
@@ -14,25 +14,27 @@
 //  for use with the TypeID template.
 //
 //===----------------------------------------------------------------------===//
-SWIFT_TYPEID_NAMED(NominalTypeDecl *, NominalTypeDecl)
-SWIFT_TYPEID_NAMED(VarDecl *, VarDecl)
-SWIFT_TYPEID_NAMED(ValueDecl *, ValueDecl)
-SWIFT_TYPEID_NAMED(ProtocolDecl *, ProtocolDecl)
-SWIFT_TYPEID_NAMED(Decl *, Decl)
-SWIFT_TYPEID_NAMED(ModuleDecl *, ModuleDecl)
-SWIFT_TYPEID(Type)
-SWIFT_TYPEID(TypePair)
+
+SWIFT_TYPEID(AncestryFlags)
+SWIFT_TYPEID(CtorInitializerKind)
 SWIFT_TYPEID(PropertyWrapperBackingPropertyInfo)
 SWIFT_TYPEID(PropertyWrapperTypeInfo)
-SWIFT_TYPEID(CtorInitializerKind)
+SWIFT_TYPEID(Requirement)
 SWIFT_TYPEID(ResilienceExpansion)
-SWIFT_TYPEID_NAMED(Optional<PropertyWrapperMutability>, PropertyWrapperMutability)
+SWIFT_TYPEID(Type)
+SWIFT_TYPEID(TypePair)
 SWIFT_TYPEID_NAMED(CustomAttr *, CustomAttr)
-SWIFT_TYPEID_NAMED(OperatorDecl *, OperatorDecl)
-SWIFT_TYPEID_NAMED(TypeAliasDecl *, TypeAliasDecl)
-SWIFT_TYPEID(AncestryFlags)
+SWIFT_TYPEID_NAMED(Decl *, Decl)
+SWIFT_TYPEID_NAMED(GenericParamList *, GenericParamList)
 SWIFT_TYPEID_NAMED(GenericSignature *, GenericSignature)
 SWIFT_TYPEID_NAMED(GenericTypeParamType *, GenericTypeParamType)
-SWIFT_TYPEID(Requirement)
 SWIFT_TYPEID_NAMED(IterableDeclContext *, IterableDeclContext)
-SWIFT_TYPEID_NAMED(GenericParamList *, GenericParamList)
+SWIFT_TYPEID_NAMED(ModuleDecl *, ModuleDecl)
+SWIFT_TYPEID_NAMED(NominalTypeDecl *, NominalTypeDecl)
+SWIFT_TYPEID_NAMED(OperatorDecl *, OperatorDecl)
+SWIFT_TYPEID_NAMED(Optional<PropertyWrapperMutability>,
+                   PropertyWrapperMutability)
+SWIFT_TYPEID_NAMED(ProtocolDecl *, ProtocolDecl)
+SWIFT_TYPEID_NAMED(TypeAliasDecl *, TypeAliasDecl)
+SWIFT_TYPEID_NAMED(ValueDecl *, ValueDecl)
+SWIFT_TYPEID_NAMED(VarDecl *, VarDecl)
```

Comment:
Seems like we're sorting by the wrong thing here. Can we flip the parameters while you're here?

Reply:
Flipping the parameters is good.  I’ll probably come back through here as part of the evaluator ergonomics patches I have to merge.